### ISSUE TITLE
Move import sorting to JuliaFormatter.jl?

### ISSUE THREAD

**Issue by MilesCranmer** [julia-vscode/LanguageServer.jl#1262]
@fredrikekre @pfitzseb I was wondering if the code action implemented in https://github.com/julia-vscode/LanguageServer.jl/pull/1076 could instead be moved to JuliaFormatter.jl, and then called via LanguageServer.jl? Having it implemented in LanguageServer makes it a bit tricky to automate things – like that feature it in a pre-commit hook or in CI testing.

**Comment by fredrikekre** [comment]
I thought that was already part of the formatter (too).

**Comment by MilesCranmer** [comment]
I don't think so, sadly. It's in a few of the style guides but you still need to perform the sorting manually. As far as I can tell you have written the only implementation of an import sorter.
